nvflare/fuel/hci/client/cli.py

```diff
@@ -118,6 +121,25 @@ def __init__(
             poc=poc,
         )
 
+        self._session_monitor = threading.Thread(target=self._check_session, args=())
+        self._session_monitor.daemon = True
+
+    def _check_session(self):
+        session_active = True
+        while session_active:
+            time.sleep(60)
+            res = self.api.check_session()
+
+            for item in res["data"]:
+                item_type = item["type"]
+                if item_type == "string":
+                    if item["data"] == "REJECT":
+                        session_active = False
+                elif item_type == "error":
+                    self.write_error(item["data"])
```

Comment:
if this suddenly shows up in admin console would users be confused about what the message is for?

Reply:
This error would only be present for the response of check_session() if the session timed out.